wait-for-idle-runner/main.js

```diff
@@ -0,0 +1,33 @@
+const { Octokit } = require("@octokit/action");
+
+const octokit = new Octokit();
+
+const NUMBER_OF_ATTEMPTS = 3000;
+const TIME_BETWEEN_ATTEMPTS_SECONDS = 5000;
+
+async function main() {
+  for (var i=0; i < NUMBER_OF_ATTEMPTS; i++) {
+    const result = await octokit.request(`GET /repos/${process.env.REPOSITORY_NAME}/actions/runners`)
+
+    // Select the runner based on its name
+    runner = result.data["runners"].filter(function(runner) {
+      return runner.name == process.env.RUNNER_NAME;
+    })[0]
+
+    if (typeof runner == "undefined") {
+      // Return if the runner does not exist
+      console.log("::set-output name=result::Runner does not exist");
+      break;
```

Comment:
Can we just `return` at this point?

Then, it might be a good idea to put a final `console.log` at the end of the function that sets the output to `Timed out` or something like that.